doto-frontend/src/components/pages/Settings/SettingsPage.js

```diff
@@ -123,40 +113,27 @@ const SubmitButton = () => {
 };
```

Comment:
Can you also get rid of the save button as it has no effect anymore? The theme and all other data changes are being applied from its relevant components.